src/ReactCursorPosition.js

```diff
@@ -430,14 +430,10 @@ export default class extends React.Component {
         return e.touches[0];
     }
 
-    isReactComponent(reactElement) {
```

Comment:
Hi @jacekradko - What is your opinion of keeping the method but changing the name to`getIsReactComponent` to avoid name collision? I like the method because its name expresses its purpose.

Reply:
Sounds good to me. Conveying intent is important.